tests/playbooks/roles/dns-records/tasks/main.yml

```diff
@@ -1,10 +1,13 @@
 ---
 
 - name: Check if a DNS record exist
+  changed_when: false
   shell: >-
     dig +noall +answer -t {{ records.type }}
-    @127.0.0.1 {{ name }} | grep '^{{ name }}'
+    @127.0.0.1 {{ record.name }}
+    | grep '^{{ record.name }}'
+    | grep -E '{{ record.value }}'
```

Comment:
I believe this can be postponed, I suspect DNSSEC deployment worked or is broken, but I am not sure why it would be half deployed. I don't mind a more readable test, later, though.

Reply:
I agree on both the necessity and the readability.